cmd/influx/main.go

```diff
@@ -56,6 +56,8 @@ func main() {
 	fs.IntVar(&c.ImporterConfig.PPS, "pps", defaultPPS, "How many points per second the import will allow.  By default it is zero and will not throttle importing.")
 	fs.StringVar(&c.ImporterConfig.Path, "path", "", "path to the file to import")
 	fs.BoolVar(&c.ImporterConfig.Compressed, "compressed", false, "set to true if the import file is compressed")
+	fs.StringVar(&c.ImporterConfig.RetentionPolicy, "retention-policy", "", "Writes the import data to the specified retention policy when using -import.")
```

Comment:
for consistency, should it be `destination-retention-policy` to match `destination-database`?